dart/lib/src/enricher/io_enricher_event_processor.dart

```diff
@@ -31,10 +31,15 @@ class IoEnricherEventProcessor extends EventProcessor {
         ? null
         : _getDevice(event.contexts.device);
 
+    final culture = _options.platformChecker.hasNativeIntegration
+        ? null
```

Comment:
Wondering if this has a bug, which is:
If I do `captureEvent(event)` and my event has `culture` data added manually and `hasNativeIntegration` is true, it will replace the current `culture` to `null` or it will keep the given `culture`?

Reply:
No, this is not a bug since the `copyWith` implementation does not override properties when you're passing `null`. See https://github.com/getsentry/sentry-dart/blob/e93430abb2ac24b30bb175ad3a54cb80d7671262/dart/lib/src/protocol/contexts.dart#L258